template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Pages/Account/Manage/Index.cshtml.cs

```diff
@@ -113,15 +116,17 @@ public class InputModel
                 return Page();
             }
 
-            var user = await _userManager.GetUserAsync(HttpContext.User);
+            var user = await _userManager.GetUserAsync(User);
             if (user == null)
             {
                 return RedirectToPage("/Error");
             }
 
             var code = await _userManager.GenerateEmailConfirmationTokenAsync(user);
-            var callbackUrl = Url.EmailConfirmationLink(user.Id, code, Request.Scheme);
-            await _emailSender.SendEmailConfirmationAsync(Input.Email, callbackUrl);
+            var userId = await _userManager.GetUserIdAsync(user);
```

Comment:
Yeah its not a big deal either way, you'd assume any app that let them change the username, would also clear the cookie/sign them in again as part of the change name flow.  That's what our manage actions do already...